### PR TITLE
Added option commitID for deploy pipeline job

### PR DESCRIPTION
**Pull request by satr**

### CODE REVIEW

File: cmd/createDeployPipelineJob.go [hunk 1]
```diff
@@ -101,12 +110,27 @@ Examples:
...
+	return nil
+}
+
 func init() {
 	createJobCmd.AddCommand(createDeployPipelineJobCmd)
 	createDeployPipelineJobCmd.Flags().StringP("application", "a", "", "Name of the application to deploy")
 	createDeployPipelineJobCmd.Flags().StringP("environment", "e", "", "Target environment to deploy in ('prod', 'dev', 'playground')")
 	createDeployPipelineJobCmd.Flags().StringP("user", "u", "", "The user who triggered the deploy")
 	createDeployPipelineJobCmd.Flags().StringToStringP("image-tag-name", "t", map[string]string{}, "Image tag name for a component: component-name=tag-name. Multiple pairs can be specified.")
+	createDeployPipelineJobCmd.Flags().StringP("commitID", "i", "", "Commit id")
```
Comment (Richard87) [inline cmd/createDeployPipelineJob.go:133]: Write what it will do, and what it requires? "Optionally add a 40 character commit id to the deployment"?

**Review by Richard87 — CHANGES_REQUESTED** [review]

File: cmd/createDeployPipelineJob.go (hunk 1 above)
Comment (Richard87) [inline cmd/createDeployPipelineJob.go:133]: Write what the commitId do and requires? "An optional 40 character commit id to tag the new pipeline job

**Review by Richard87 — CHANGES_REQUESTED** [review]

File: cmd/createDeployPipelineJob.go (hunk 1 above)
Comment (satr) [reply cmd/createDeployPipelineJob.go]: Fixed

**Review by Richard87 — APPROVED** [review]
lgtm

**Pull request merged**